modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/utils/DeployEnvironments.java

```diff
@@ -17,7 +17,7 @@
 import org.eclipse.xpanse.modules.models.service.common.enums.Csp;
 import org.eclipse.xpanse.modules.models.service.deploy.exceptions.FlavorInvalidException;
 import org.eclipse.xpanse.modules.models.servicetemplate.DeployVariable;
-import org.eclipse.xpanse.modules.models.servicetemplate.Flavor;
+import org.eclipse.xpanse.modules.models.servicetemplate.FlavorWithProperties;

```

Comment:
I feel like keeping the original `Flavor` is better, with adding a new class named `FlavorWithoutProperties`/`FlavorBasic`.

Reply:
Ok. I added  a new class `FlavorBasic` without the field `properties` for user querying.